tests/test_templatetags.py

```diff
@@ -364,11 +356,7 @@ def test_nested_component_templates_all_shown_as_used(self):
 
 class NestedSlotTests(SimpleTestCase):
     class NestedComponent(component.Component):
-        def context(self):
-            return {}
```

Comment:
Is this safe to remove?

Reply:
Yep. Will run through the code once more to check everything. The most important part is for tests to pass anyways :D